docs/reference/binary_quadratic_model.rst

```diff
@@ -81,8 +81,6 @@ Energy
 Conversion Functions
 ====================
 
-.. automodule:: dimod.binary_quadratic_model_convert
-
 .. currentmodule:: dimod
 
 .. autosummary::
```

Comment:
Did you try building the docs? I think these all need to become `BinaryQuadraticModel.to_ising` etc